code/src/App.js

```diff
@@ -1,12 +1,33 @@
 import React from 'react'
+import { Albums } from './components/Albums.js'
+import { Playlist } from './components/Playlists.js'
```

Comment:
It's completely fine to use `.js` in imports, but generally, best practice is to exclude it. So just `./components/Playlists` :)